imports/plugins/core/orders/client/templates/orders.js

```diff
@@ -303,6 +316,10 @@ Template.orderStatusDetail.helpers({
     return i18next.t("orderShipping.noTracking");
   },
 
+  orderStatus: function () {
```

Comment:
We want to stay consistent with the use of `arrow function` vs `anonymous function`